modules/presence/hash.c

```diff
@@ -793,4 +793,48 @@ int delete_cluster_query(str* pres_uri, int event, unsigned int hash_code)
 	return 0;
 }
 
+/* 
+ * entry must be locked before calling this function 
+ * Returns 1 if 
+            - entry exisits subscription is expired. 
+			- entry does not exist.
+ * Returns 0 if entry is found and subscription is active. 
+*/
+int is_sub_expired(shtable_t htable,unsigned int hash_code,str to_tag)
```

Comment:
Not urgent but would be nice to format code additions (space between names, space before and after operators, etc) to match the style-guide of upstream

Provided a few examples but non-blocking
```suggestion
int is_sub_expired(shtable_t htable, unsigned int hash_code, str to_tag)
```